Makefile

```diff
@@ -1,36 +1,36 @@
-.PHONY: all
-
-version_file := src/bootstrap4/_version.py
-version := $(word 3, $(shell cat ${version_file}))
-
-version:
-	@echo $(version)
+.PHONY: clean test tox reformat lint docs build publish
 
 clean:
 	rm -rf build dist *.egg-info
 
 test:
-	python runtests.py
+	coverage run manage.py test
```

Comment:
You usually want to add a `--source=src` flag here, to prevent coverage from adding dependencies and tests to the coverage report. It used to be (incorrectly) configured in `.coveragerc`, but that file was removed.

Reply:
The configuration is (correct) in `tox.ini`.

```
[coverage:run]
source = src/bootstrap4

[coverage:report]
show_missing = True
skip_covered = True
```

Alas, tox itself cannot use that one so it's hardcoded in the tox test command.